tools/build-companion.sh

```diff
@@ -20,42 +20,45 @@ if [ "$3" == "" ]; then
    exit 1
 fi
 
+if [ "$JOBS" == "" ]; then
+    JOBS=2
+fi
```

Comment:
I can also remove that from this pull request for now until we have a better option